packages/harmony/src/types/colors.ts

```diff
@@ -39,3 +39,5 @@ export type ColorValue =
   | 'lightGreen'
   | 'gradient'
   | 'none'
+
+export type BackgroundColorValue = 'default' | 'surface1' | 'surface2' | 'white'
```

Comment:
Is something like this available on main rn inside semantic colors file?

Reply:
yeah I hit this during the merge. Already updated!